core/lib/devise_extension.rb

```diff
@@ -29,6 +30,7 @@ module Controllers
 
 # modules
 Devise.add_module :password_expirable, controller: :password_expirable, model: 'devise/models/password_expirable', route: :password_expired
+Devise.add_module :session_limitable, model: 'devise/models/session_limitable'
```

Comment:
This one needs to be removed to?

Good to rebase and push to QA otherwise

Reply:
Good catch